src/open_producten/producttypen/serializers/jsonschema.py

```diff
@@ -0,0 +1,21 @@
+from django_json_schema_model.models import JsonSchema
+from jsonschema import Draft202012Validator
+from jsonschema.exceptions import SchemaError
+from rest_framework import serializers
+
+
+class JsonSchemaSerializer(serializers.ModelSerializer):
+
+    schema = serializers.DictField()
+
+    def validate_schema(self, schema):
+        try:
+            Draft202012Validator.check_schema(schema)
+        except SchemaError as e:
+            raise serializers.ValidationError(e.message)
+
+        return schema
+
+    class Meta:
+        model = JsonSchema
+        fields = "__all__"
```

Comment:
Same comment as above: it's better to define serializer fields explicitly. For example here it's not obvious is PK included in the serializer